entrypoint.sh

```diff
@@ -50,5 +51,20 @@ fi
 
 export OKTETO_DISABLE_SPINNER=1
 
-echo running: okteto pipeline deploy --name "${name}" --branch="${branch}" --repository="${GITHUB_SERVER_URL}/${repository}" ${params} --wait
-okteto pipeline deploy --name "${name}" --branch="${branch}" --repository="${GITHUB_SERVER_URL}/${repository}" ${params} --wait
+output=$(cat <<-END
```

Comment:
Could you leave it directly running the command? you can use `\` to break lines for clarity